pycket/test/testhelper.py

```diff
@@ -4,27 +4,60 @@
 # Assorted test helpers
 #
 import os
+from tempfile import NamedTemporaryFile, gettempdir
 from pycket.expand import expand, to_ast, expand_string, parse_module
 from pycket.pycket_json import loads
 from pycket.interpreter import *
 from pycket.env import ToplevelEnv
 from pycket import values
 
+import pytest
+
 #
 # basic runners
 #
 
 # This is where all the work happens
 
+delete_temp_files = True
+
 def run_ast(ast):
     env = ToplevelEnv()
     env.globalconfig.load(ast)
     mod = interpret_module(ast, env)
     return mod
 
+def expand_from_bytecode(m, srcloc):
+    f = NamedTemporaryFile(delete=delete_temp_files)
```

Comment:
You can use a `with` statement on `NamedTemporaryFile` objects. That way we ensure all our temp files are cleaned up properly.